lib/global/index.js

```diff
@@ -6,11 +6,22 @@
 'use strict'; /* eslint strict:0 */
 
 var getGlobal = function () {
-  if (typeof globalThis !== 'undefined') { return globalThis; }
-  if (typeof window !== 'undefined') { return window; }
-  if (typeof self !== 'undefined') { return self; }
-  if (typeof global !== 'undefined') { return global; }
-  throw new Error('unable to locate global object');
+  var globalObj = (function () {
+    if (typeof globalThis !== 'undefined') { return globalThis; }
+    if (typeof window !== 'undefined') { return window; }
+    if (typeof self !== 'undefined') { return self; }
+    if (typeof global !== 'undefined') { return global; }
+    throw new Error('unable to locate global object');
+  })()
+  /*
+    The below code was added in case there is pollution in global namespace
+    windows object during transpilation with __esModule being set. Below code would support
+    global import in all bundle use cases
+  */
+  if (globalObj && globalObj.__esModule) {
+    globalObj.default = Object.assign({}, globalObj)
```

Comment:
Can we find a way to not pollute the global `window` object and just have the `default` property set to the exported reference object.